docs/tutorial/authentication-tutorial.md

```diff
@@ -256,19 +256,16 @@ To check if a user can access the content, you can wrap the restricted content i
 import React, { Component } from "react"
 import { navigate } from "gatsby"
 import { isLoggedIn } from "../services/auth"
-class PrivateRoute extends Component {
-  componentDidMount() {
-    const { location } = this.props
-    let noOnLoginPage = location.pathname !== `/app/login`
-    if (!isLoggedIn() && noOnLoginPage) {
-      navigate("/app/login")
-      return null
-    }
-  }
-  render() {
-    const { component: Component, ...rest } = this.props
-    return <Component {...rest} />
+
+const PrivateRoute = ({ component: Component, location, ...rest }) => {
```

Comment:
remove the let expression to make it act in accordance with the provided example in the repo

Reply:
Ahh! Done.

Reply:
👍